_includes/source_code/code/37-PDP/samepizzas/samepizzas_subtask2.cc

```diff
@@ -0,0 +1,22 @@
+#include <algorithm>
+#include <cstdio>
+#include <vector>
+
+int main(){
+   FILE *in = fopen("samepizzas.in", "r");
+   FILE *out= fopen("samepizzas.out", "w");
+   long P,N,K;
+   fscanf(in, "%ld\n", &P);
```

Comment:
Στην fscanf αν βαλεις 1 κενο πριν τα argument descriptots, αγνοεί white spaces (και τα enter) μέχρι να φτάσει στους χρήσιμους χαρακτήρες. Tο \n στα test cases ίσως να παραείναι strict περιορισμός και ψάχνει το enter

``fscanf(in, " %ld", &P);``